pkg/common/consumer/consumer_manager.go

```diff
@@ -111,6 +134,38 @@ func NewConsumerGroupManager(logger *zap.Logger, serverHandler controlprotocol.S
 	return manager
 }
 
+// AddNotification creates a new ManagerEvent channel and returns it.  Manager events will be
+// broadcast to all channels created this way.
+func (m *kafkaConsumerGroupManagerImpl) AddNotification() <-chan ManagerEvent {
+	eventChan := make(chan ManagerEvent)
```

Comment:
`AddNotification()` seems like a weird name for this?  Maybe `RegisterForNotifications()` or `ReceiveNotifications()` or `GetNotificationChannel()` or something?  Not a big deal - up to you - just a suggestion.

Reply:
I like GetNotificationChannel; I'll use that.  I think you're right that the consumer of the interface doesn't care that it's adding a channel to a list internally.